mirrord/operator/src/client.rs

```diff
@@ -735,22 +753,47 @@ impl OperatorApi<PreparedClientCert> {
         target: Target,
         scale_down: bool,
         namespace: &str,
-        split_queues: &SplitQueuesConfig,
+        split_queues: Option<SplitQueuesConfig>,
     ) -> OperatorApiResult<CopyTargetCrd> {
-        let name = TargetCrd::urlfied_name(&target);
-
-        let requested = CopyTargetCrd::new(
-            &name,
-            CopyTargetSpec {
-                target,
-                idle_ttl: Some(Self::COPIED_POD_IDLE_TTL),
-                scale_down,
-                split_queues: Some(split_queues.clone()),
-            },
-        );
+        let user_id = self.get_user_id_str();
+
+        let copy_target_api: Api<CopyTargetCrd> = Api::namespaced(self.client.clone(), namespace);
+
+        let existing_copy_targets =
+            copy_target_api
+                .list(&ListParams::default())
```

Comment:
This might be a nitpick, but the operator could set a label with the user_id, and we could set a label selector and only get the right copy target, instead of getting all of them from the API and then filtering.
This could be an issue for later if you do think it's a good optimisation but it's not important for now (but adding the label is probably easy so I think we can do the operator side already).